test/functional/api/v1/dependencies_controller_test.rb

```diff
@@ -78,23 +78,23 @@ class Api::V1::DependenciesControllerTest < ActionController::TestCase
       result = [
         {
           'name'              => 'myrails',
-          'number'            => '3.0.0',
+          'number'            => '1.0.0',
           'platform'          => 'ruby',
           'rubygems_version'  => '>= 2.6.3',
           'ruby_version'      => '>= 2.0.0',
           'checksum'          => 'tdQEXD9Gb6kf4sxqvnkjKhpXzfEE96JucW4KHieJ33g=',
-          'created_at'        => '2016-05-24T00:00:00.000Z',
+          'created_at'        => '2016-05-24 00:00:00',
```

Comment:
This is closer but still not exactly what the current format is.
Currently I'm seeing: `2016-05-24 00:00:00 +0000`